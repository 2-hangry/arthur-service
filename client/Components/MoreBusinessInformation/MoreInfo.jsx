import React from "react";
import InformationString from "./InformationString.jsx";
import styles from '../../styles.css.js';

const MoreInfo = props => {
  const infoArr = Object.entries(props.moreInfo);
  return (
    <div>
      <p style={styles.sectionTitle}>More Business Info</p>
      <div>
        {infoArr.map(info => {
          return <InformationString info={info} />;
        })}
      </div>
    </div>
  );
};

export default MoreInfo;
