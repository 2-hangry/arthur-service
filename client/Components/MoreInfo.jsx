import React from "react";
import InformationString from "./InformationString.jsx";
const divStyle = {
  color: "red",
  fontFamily: "sans-serif",
  fontSize: 14+'px'
};


const MoreInfo = props => {
  const infoArr = Object.entries(props.moreInfo.more_info);

  return (
    <div>
      <h3 style={divStyle}>More business info</h3>
      <div>
        {infoArr.map(info => {
          return <InformationString info={info} />;
        })}
      </div>
    </div>
  );
};

export default MoreInfo;
