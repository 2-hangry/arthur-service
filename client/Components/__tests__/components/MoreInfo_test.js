


import React from "react";
import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import MoreInfo from "../../MoreBusiessInformation/MoreInfo.jsx";

Enzyme.configure({ adapter: new Adapter() });
import renderer from "react-test-renderer";

it("renders correctly", () => {
    const props = {moreInfo: {more_info: {'Accepts Apple Pay':
        false,
        'Accepts Credit Cards':
        false,
        'Accepts Google Pay':
        false,
        'Alcohol':
        true,
        'Attire Casual':
        "sloppy",
        'Bike Parking':
        false,
        'Caters':
        true,
        'Delivery':
        true,
        'Gender Neutral Restroom':
        false,
        'Good For':
        "breakfast",
        'Good for Groups':
        true,
        'Good for Kids':
        false,
        'Has Soy-free Options':
        true,
        'Has TV':
        true,
        'Liked by Vegetarians':
        false,
        'Noise Level Average':
        "loud",
        'Outdoor Seating':
        true,
        'Parking Street':
        false,
        'Take-out':
        false,
        'Takes Reservations':
        false,
        'Waiter Service':
        false,
        'Wheelchair Accessible':
        false,
        'Wi-Fi':
        false
      
    }}}
    

  const tree = renderer.create(<MoreInfo {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});