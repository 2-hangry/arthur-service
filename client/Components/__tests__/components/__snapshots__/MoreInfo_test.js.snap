// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <p
    style={
      Object {
        "color": "red",
        "fontWeight": "bold",
      }
    }
  >
    More Business Info
  </p>
  <div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Accepts Apple Pay
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         No
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Accepts Credit Cards
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         No
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Accepts Google Pay
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         No
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Alcohol
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         Yes
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Attire Casual
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         
        sloppy
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Bike Parking
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         No
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Caters
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         Yes
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Delivery
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         Yes
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Gender Neutral Restroom
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         No
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Good For
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         
        breakfast
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Good for Groups
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         Yes
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Good for Kids
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         No
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Has Soy-free Options
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         Yes
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Has TV
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         Yes
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Liked by Vegetarians
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         No
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Noise Level Average
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         
        loud
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Outdoor Seating
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         Yes
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Parking Street
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         No
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Take-out
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         No
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Takes Reservations
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         No
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Waiter Service
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         No
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Wheelchair Accessible
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         No
      </span>
    </div>
    <div
      style={
        Object {
          "fontSize": 13,
        }
      }
    >
      <br />
      <span>
        Wi-Fi
      </span>
      <span
        style={
          Object {
            "fontWeight": "bold",
          }
        }
      >
         No
      </span>
    </div>
  </div>
</div>
`;
